### DISCUSSION TITLE
How can I input while debugging?

### DISCUSSION THREAD

**Discussion by yvvt0379** [microsoft/vscode-cpptools#9363] [Q&A]
Assume I have a C/C++ program that requires user input, how can I input data from keyboard while debugging?

**Comment by atifkarim** [comment]
I have tested in  `VSCode`

- In `terminal` window of the VSCode put the required value (user input) while reaches to the `std::cin >> variable_name;` line 
- Press the `step over` or `step into` button of the debugger tool
- Continue
